config/rubocop.yml

```diff
@@ -107,10 +107,31 @@ Style/StringMethods:
 Rails/SaveBang:
   Enabled: true
 
+Rails/SkipsModelValidations:
+  Blacklist:
+    # - update_all
+    - decrement!
+    - decrement_counter
+    - increment!
+    - increment_counter
+    - toggle!
+    - touch
+    - update_attribute
+    - update_column
+    - update_columns
+    - update_counters
+
 # Don't restrict some cops to certain directories:
 
 Rails/FindBy:
   Include: null
 
 Rails/FindEach:
   Include: null
+
+# brakeman currently cannot parse squiggly heredocs(i.e. <<~DOC)
+Layout/IndentHeredoc:
+  Enabled: false
+
+RSpec/MessageSpies:
+  Enabled: false
```

Comment:
I didn't dare enabling this as it would mean re-writing all of the specs to replace mocks with spies.